Tools/TestWebKitAPI/Tests/WebKitCocoa/WKPDFView.mm

```diff
@@ -191,7 +191,7 @@ static void createHostViewForExtensionIdentifier(void(^callback)(id hostViewCont
 
 #endif
 
-#if ENABLE(UI_PROCESS_PDF_HUD)
+#if ENABLE(PDFKIT_PLUGIN)
```

Comment:
@achristensen07 This one too? (this change enabled iOS to build)

Reply:
yes